### PR TITLE
doc: remove notion of leveldown-specific sync option

### PR DESCRIPTION
**Pull request by juliangruber**
This pull request is targeted at the `v2` branch. If wished I can recreate it against `master`, shouldn't matter too much.

As in https://github.com/Level/levelup/issues/366

### CODE REVIEW

**Comment by ralphtheninja** [comment]
+1

**Comment by juliangruber** [comment]
@ralphtheninja do you think this should go into master of v2?

**Comment by ralphtheninja** [comment]
Has sync been removed from master?

**Comment by ralphtheninja** [comment]
Oh nevermind, it's leveldown specific. Then I think we could include it on master as well. Sorry for my ignorance :)

File: lib/levelup.js [hunk 1]
```diff
@@ -65,9 +65,17 @@ function LevelUP (location, options, callback) {
     options  = {}
   }
```
Comment (ralphtheninja) [inline lib/levelup.js]: @juliangruber I'm gonna do it your way here and start off by moving db to `options.db` and once that is done get rid of `location`

File: lib/levelup.js [hunk 2]
```diff
@@ -65,9 +65,17 @@ function LevelUP (location, options, callback) {
...
 
-  if ((!options || typeof options.db != 'function') && typeof location != 'string') {
+  if (!options && typeof options.db != 'function') {
```
Comment (ralphtheninja) [inline lib/levelup.js]: Going through what you did in this PR and it seems to me that this line is wrong, shouldn't it be

```js
if (!options || typeof options.db !== 'function') {}
```

Comment (ralphtheninja) [inline lib/levelup.js]: cc @juliangruber

**Pull request closed without merging**